### PR TITLE
Docker-compose file was created.

### PR DESCRIPTION
**Pull request by Ilya-Chudinovskikh**

### CODE REVIEW

**Comment by Ilya-Chudinovskikh** [comment]
The command I used: docker-compose -f "C:\Users\Илья\source\repos\WordOfTheDayApp\docker-compose.yml" up -d --build

**Comment by Ilya-Chudinovskikh** [comment]
Now it seems to work correctly. As I found, the problem with the running of the database was related to the password in the docker-compose file.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  ms-sql-server:
+    image: mcr.microsoft.com/mssql/server:2019-latest
+    environment:
+      ACCEPT_EULA: "Y"
+      SA_PASSWORD: "Password123!"
+      MSSQL_PID: Express
+    ports:
+      - 1433:1433
+    volumes:
+      - //c/Users/Илья/WordContext-03daec14-a505-48b7-8fb2-72bc85eb0fcd
```
Comment (php5art) [inline docker-compose.yml]: Can you rewrite definition of volume to not depend on your local path? Check this documentation https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes

**Review by php5art — CHANGES_REQUESTED** [review]

File: Scripts/WordOfTheDay.Install.ps1 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿cd C:\Users\Илья\source\repos\WordOfTheDayApp
+dotnet-ef database update -s WordOfTheDay.Api -p WordOfTheDay.Repository
+docker-compose  -f "C:\Users\Илья\source\repos\WordOfTheDayApp\docker-compose.yml" build
```
Comment (php5art) [inline Scripts/WordOfTheDay.Install.ps1]: Can you use relative path here?

File: Scripts/WordOfTheDay.Run.Dotnet.ps1 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿cd C:\Users\Илья\source\repos\WordOfTheDayApp
+dotnet build
+cd C:\Users\Илья\source\repos\WordOfTheDayApp\WordOfTheDay.Api
```
Comment (php5art) [inline Scripts/WordOfTheDay.Run.Dotnet.ps1]: Can you use relative path here?

File: Scripts/WordOfTheDay.Run.ps1 [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿cd C:\Users\Илья\source\repos\WordOfTheDayApp
+docker-compose  -f "C:\Users\Илья\source\repos\WordOfTheDayApp\docker-compose.yml" up -d --build
```
Comment (php5art) [inline Scripts/WordOfTheDay.Run.ps1]: Can you use relative paths here?

File: Scripts/WordOfTheDay.Run2.ps1 [hunk 5]
```diff
@@ -0,0 +1,3 @@
+﻿cd C:\Users\Илья\source\repos\WordOfTheDayApp
+& "C:\Dev\Scripts\WordOfTheDay.Run.ps1"
+docker-compose  -f "C:\Users\Илья\source\repos\WordOfTheDayApp\docker-compose.yml" up
```
Comment (php5art) [inline Scripts/WordOfTheDay.Run2.ps1]: Can you use relative paths here?

**Review by php5art — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (Ilya-Chudinovskikh) [reply docker-compose.yml]: Done.

File: Scripts/WordOfTheDay.Install.ps1 (hunk 2 above)
Comment (Ilya-Chudinovskikh) [reply Scripts/WordOfTheDay.Install.ps1]: Done.

File: docker-compose.yml (hunk 1 above)
Comment (Ilya-Chudinovskikh) [reply docker-compose.yml:12]: Done.

File: Scripts/WordOfTheDay.Run.ps1 (hunk 4 above)
Comment (Ilya-Chudinovskikh) [reply Scripts/WordOfTheDay.Run.ps1:2]: Done.

File: Scripts/WordOfTheDay.Run.Dotnet.ps1 (hunk 3 above)
Comment (Ilya-Chudinovskikh) [reply Scripts/WordOfTheDay.Run.Dotnet.ps1]: done

File: Scripts/WordOfTheDay.Run.ps1 (hunk 4 above)
Comment (Ilya-Chudinovskikh) [reply Scripts/WordOfTheDay.Run.ps1]: done

File: Scripts/WordOfTheDay.Run2.ps1 (hunk 5 above)
Comment (Ilya-Chudinovskikh) [reply Scripts/WordOfTheDay.Run2.ps1]: done

**Review by php5art — APPROVED** [review]

**Pull request merged**